frontend/src/lib/components/LemonModal/LemonModal.tsx

```diff
@@ -28,6 +28,11 @@ export interface LemonModalProps {
     closable?: boolean
     /** Expands the modal to fill the entire screen */
     fullScreen?: boolean
+    /**
+     * This can be used when the modal should be high enough in the z-index stack to be shown above everything
+     * except a few parts of the page chrome. See vars.scss for the z-index values
+     * */
```

Comment:
Looking at `vars.scss`, it seems to me this is specifically about popups. Is there any other conflicting component? If not, this might be clearer as `forceAbovePopups`. Still a hack, but more targeted.
The ideal solution would probably be setting a higher z-index on popups initiated from modal, and lower on others, but I definitely see how that's not the best use of our time right now.